pages/api/controllers/index.js

```diff
@@ -1,8 +1,8 @@
-import {getHomepageData} from '../services/home-page-service';
+import getHomepageData from '../services/home-page-service';
```

Comment:
Note for all controllers/services/repositories: should these be named in camel case, and imported as such? Or in snake case, but imported in camel case as the same name? Not enough standardization here